lib/line.js

```diff
@@ -4,12 +4,14 @@ module.exports = class CovLine {
     // note that startCol and endCol are absolute positions
     // within a file, not relative to the line.
     this.startCol = startCol
+    this.minCol = startCol + lineStr.length - lineStr.trimStart().length
```

Comment:
Trimming lines may actually be the cause of the remapping issues I saw in `c8`, since we specifically see breakage with tests that exercise SourceMap files.

The source map lookup logic is quite finicky, and trimming before performing it may need to be avoided.

Reply:
@bcoe the investigation I did suggests me that the tests are broken but that the accuracy of the coverage improves. As I posted [here](https://github.com/istanbuljs/v8-to-istanbul/pull/172#issuecomment-997116693) and in the node-fetch comparisons the existing logic seems to often miss the first line of a range. I look forward to your feedback on those examples